logging/Limiting_Access_to_Logs.md

```diff
@@ -15,39 +17,86 @@ only for a specified namespace.
+  
+### Logadm Access Controls
 
-## Implement Access Controls for a Tenant or Namespace
+The logadm user is the day-to-day user for log monitoring for cluster administrators. The logadm user has access to all of the log messages collected by the log-monitoring components. This means that the logadm user has access to the following messages: 
+
+- log messages from all SAS Viya namespaces and all tenants within those namespaces.
+- log messages from all non-SAS Viya namespaces including the Kubernetes infrastructure namespaces.
+ 
+Although the logadm user can access all of the log messages, logadm can access only the "cluster_admins" Kibana-tenant space.
+
+The access controls are designed to provide logadm users with full cluster access to perform the log-monitoring activities needed for their job.  Limiting these users to the "cluster_admins" Kibana-tenant space helps in the following ways:
+
+- simplifies the Kibana user experience.
+- eliminates the potential confusion that could otherwise occur if the user could  access Kibana content in the other Kibana-tenant spaces.
+
+
+| Back-end Role | Role | Purpose |
+| --- | --- | --- |
+|     | v4m_kibana_user | Grants user access to the Kibana functionality needed. (This role is not unique to the logadm user). |
+|     | search_index | Grants access to all collected log messages. |
+|     | tenant_cluster_admins | Grants access to the cluster_admins Kibana-tenant space.|
+|V4MCLUSTER_ADMIN_kibana_users |    | Provides an easy way to link the above roles with a user.
+
+Linking users to the back-end role of V4MCLUSTER_ADMIN_kibana_users via the Kibana security plug-in grants those users the access describe above.
+
+You can create new users and link them to these access controls by running the user.sh script.  For example, to create a user called "Chen" and link her to these access controls, submit the following command:
+
+<pre>
+/logging/bin/user.sh CREATE --user <i>username</i> --password <i>password</i> --namespace _all_ --tenant _all_
+</pre>
+
+Where: 
+
+- *username* is chen.
+- *password* is the password to WHAT?.
+
+## <a name="implement_access"></a>Implement Access Controls for a Tenant or Namespace
 
 Use the `/logging/bin/onboard.sh` script to implement logging access 
 controls for a namespace or SAS Viya tenant. The script performs these actions: 
 
-- Creates a Kibana tenant space
+- Creates a Kibana-tenant space.
 - Loads Kibana content such as visualizations and dashboards into the Kibana 
-tenant space
+tenant space.
 - Creates the access controls to limit access to only the appropriate 
-back-end data
-- Links the access controls to the Kibana tenant space
-- Creates a new user that can access only the information in this 
-Kibana tenant space (optional)
+back-end data.
+- Links the access controls to the Kibana-tenant space.
+- (Optional) Creates a new user that can access only the information in this 
+Kibana-tenant space.
```

Comment:
Why have periods for sentence fragments?

Reply:
There is an introductory clause that makes them complete sentences.
FYI, I am running the markdown content through a grammar checker that we use over in Pubs. (David wasn't.)